templates/f5/1/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+external-lb:
+  image: rancher/external-lb:v0.3.6
+  command: -provider=f5_BigIP
+  expose: 
+   - 1000
+  environment:
+    F5_BIGIP_HOST: ${F5_BIGIP_HOST}
+    F5_BIGIP_USER: ${F5_BIGIP_USER}
+    F5_BIGIP_PWD: ${F5_BIGIP_PWD}
+    LB_TARGET_RANCHER_SUFFIX: ${LB_TARGET_RANCHER_SUFFIX}
+    CATTLE_URL: ${CATTLE_URL}
+    CATTLE_ACCESS_KEY: ${CATTLE_ACCESS_KEY}
+    CATTLE_SECRET_KEY: ${CATTLE_SECRET_KEY}
```

Comment:
These env variables ares added automatically with labels, don't need to add them manually, could you please remove them?

```
-    CATTLE_URL: ${CATTLE_URL}
-    CATTLE_ACCESS_KEY: ${CATTLE_ACCESS_KEY}
-    CATTLE_SECRET_KEY: ${CATTLE_SECRET_KEY}
```

Reply:
updated